pkg/extract/types/detection/detection.go

```diff
@@ -104,6 +105,8 @@ func (t RangeType) String() string {
 		return "pypi"
 	case RangeTypeMaven:
 		return "maven"
+	case RangeTypeFreeBSDPkg:
+		return "freebsd"
```

Comment:
The order is OS pkg manager, library, so I think it would be better to go under `RangeTypePacman`.

Reply:
done!